platform/src/ActivityManager.js

```diff
@@ -252,25 +251,9 @@ class ActivityManager {
      * Fetches the contents of the activity with the provided ID
      */ 
     fetchActivity(id) {
-        if (!this.hasActivity(id)) {
-            var xhr = new XMLHttpRequest();
```

Comment:
where is this fetching happening now?

Reply:
The removed code only loaded stored activities when following a previously generated Epsilon Playground link. So the activity loading behaviour for us is unchanged as we now use github as a mechanism for storing user changes.

The actual activities are loaded on Line 40 of platform/src/ActivityManager.js